lib/ddr/datastreams/fits_datastream.rb

```diff
@@ -8,17 +8,19 @@ class FitsDatastream < ActiveFedora::OmDatastream
       t.root(path: "fits",
              xmlns: FITS_XMLNS,
              schema: FITS_SCHEMA)
+      t.version(path: {attribute: "version"})
```

Comment:
Do you think it's clear enough that this is the fits version?  or should the term be fits_version?  I see you have format_version below so maybe this is fine as it is.